BluetoothLEExplorer/BluetoothLEExplorer/Views/SettingsPage.xaml

```diff
@@ -79,12 +79,18 @@
                                       OffContent="Dark theme" OnContent="Light theme"
                                       RelativePanel.AlignLeftWithPanel="True"
                                       RelativePanel.Below="UseShellDrawnBackButtonToggleSwtich" />
-                        
+
                         <ToggleSwitch x:Name="UseCachingToggleSwitch" Header="Use Caching"
                                       IsOn="{Binding UseCachingButton, Mode=TwoWay}"
                                       OffContent="Will make over the air requests if necessary based on GattRobustCaching if supported." OnContent="Will use locally cached values where possible."
                                       RelativePanel.AlignLeftWithPanel="True"
                                       RelativePanel.Below="UseLightThemeToggleSwitch" />
+
+                        <ToggleSwitch x:Name="CloseConnectionsToggleSwitch" Header="Close Connections"
+                                      IsOn="{Binding CloseConnectionsButton, Mode=TwoWay}"
+                                      OffContent="Leave Bluetooth connections open while navigating the app." OnContent="Close Bluetooth conections upon navigating to Discovery page."

```

Comment:
>the app [](http://example.com/codeflow?start=100&length=7)

maybe

"back to the Discovery page"

?

Reply:
Personally I like navigating to the Discovery page better because I think of the app as more of a graph then a stack.

Reply:
I think users understand that there is a "stack" for back